android/settings.gradle

```diff
@@ -13,3 +13,16 @@ localPropertiesFile.withReader("UTF-8") { reader -> properties.load(reader) }
 def flutterSdkPath = properties.getProperty("flutter.sdk")
 assert flutterSdkPath != null, "flutter.sdk not set in local.properties"
 apply from: "$flutterSdkPath/packages/flutter_tools/gradle/app_plugin_loader.gradle"
+def flutterProjectRoot = rootProject.projectDir.parentFile.toPath()
```

Comment:
what's the usage?

Reply:
Good question, I didn't saw the change. Will remove it!

Reply:
after this, seems good to accept